lib/services/lookup.js

```diff
@@ -360,6 +364,24 @@ function lookupServiceFactory(
         });
     };
 
+    /**
+     * Returns the proxy server for the given Node ID.
+     * @param  {String} id Node ID
+     * @return {Promise.<String>} A promise fulfilled to the Proxy server URL
+     * that corresponds to this node
+     */
+    LookupService.prototype.nodeIdToProxy = function (id) {
+        assert.isMongoId(id);
```

Comment:
We're trying to remove references to `isMongoId` as we start to add support for other databases.